src/Native/gc/env/gcenv.base.h

```diff
@@ -96,7 +96,7 @@ inline HRESULT HRESULT_FROM_WIN32(unsigned long x)
 #define UNREFERENCED_PARAMETER(P)          (void)(P)
 
 #ifdef PLATFORM_UNIX
-#define  _vsnprintf vsnprintf
+#define _vsnprintf_s(string, sizeInBytes, count, format, args) vsnprintf(string, sizeInBytes, format, args)
```

Comment:
Looking at the usage it seems like `args` is always a `va_list` (and that seems consistent with the Windows CRT definition of _vsnprintf_s).